client/onboarding/kyc/index.tsx

```diff
@@ -10,9 +10,9 @@ import WooLogo from 'assets/images/woo-logo.svg';
 import Page from 'components/page';
 import { OnboardingContextProvider } from 'onboarding/context';
 import EmbeddedKyc from 'onboarding/steps/embedded-kyc';
-import strings from 'onboarding/strings';
 import { getConnectUrl } from 'utils';
 import { trackKycExit } from 'wcpay/onboarding/tracking';
+import CloseIcon from 'assets/images/icons/close.svg';
```

Comment:
We should avoid using SVG-in-JS, see this P2 post for more details on it: pdaBTz-lC-p2 

You can instead use something like this:

```react
<img src={ 'assets/images/icons/close.svg'  } alt="Close" />
```

In WC Core, we have a constant `WC_ASSET_URL` that makes this easier, but I'm not sure if we have the same for WooPayments.

Reply:
Another P2 link that may be useful: pe4Hk0-1d6-p2#comment-1459

Reply:
Actually, we can use the import in the WooPayments repo by appending `?asset` to the end of the URL. This prevents it from being included in the build file. Ismael implemented this functionality.

```suggestion
import CloseIcon from 'assets/images/icons/close.svg?asset';
```

Reply:
thanks @mordeth !